src/content/whats-new/2022/01/whats-new-01-12-core-user-type.md

```diff
@@ -0,0 +1,48 @@
+---
+title: 'Introducing our new core user type
+' 
+summary: 'Unlock telemetry data access for even more developers in your org
+' 
+releaseDate: '2022-01-12' 
+learnMoreLink: 'https://example.com' 
+getStartedLink: 'https://example.com'
+---
+
+Observability is better together. Our new core user type enables more developers than ever to understand code performance better, collaborate and prioritize work directly from their IDE, and access relevant error details including stack traces, logs, alerts, and more.
+
+**New core users enable developers to:** 
+* Optimize code performance and feature planning with access to telemetry data from production and pre-production environments directly in their IDE via the [New Relic CodeStream integration.](https://newrelic.com/codestream) 
+
+* Collaborate with DevOps, site reliability engineers (SREs), and ops engineers with a direct path from [error tracking](https://newrelic.com/platform/errors-inbox) in New Relic One to the relevant code block directly in their IDE.
+* Detect patterns and outliers in [log data](https://one.newrelic.com/launcher/logger.log-launcher?platform[accountId]=3386328&platform[timeRange][duration]=1800000&platform[$isFallbackTimeRange]=true&state=18f8ab72-f351-81d7-8b9d-d0a44b5b94a3) to resolve issues and improve code performance. 
+* Build and run [custom applications](https://docs.newrelic.com/docs/new-relic-one/use-new-relic-one/build-new-relic-one/build-custom-new-relic-one-application/) in New Relic One to surface performance insights, optimize cloud costs, and more. 
+Onboard new developers with observability best practices and shared baselines for metrics.
+
+
+## Preview Period: Try it core user capabilities today
+
+All existing basic users currently have free access to core user capabilities through Feb. 28, 2022, as part of a preview period. After the preview period, only full platform or core users can access the New Relic CodeStream integration, errors inbox, New Relic One custom apps, and advanced log management features.
+
+**To start viewing error tracking and other telemetry data directly within your IDE,** follow the [instructions](https://newrelic.com/codestream) to set up the New Relic * CodeStream integration.
+
+**To enable errors inbox** at your account, follow these steps. Afterwards, errors groups will appear in your inbox.
+From [one.newrelic.com](one.newrelic.com), select errors inbox from the top nav.
+
+* If this is your first time accessing [errors inbox](https://one.newrelic.com/launcher/errors-inbox.launcher?platform[accountId]=3386328&platform[timeRange][duration]=86400000&platform[$isFallbackTimeRange]=true&state=49e6d628-79ac-7022-1689-a6a6b7673f54), you will be prompted to select a workload in the top left.
+* If you have no workloads set up, you will be prompted to create one before you can use errors inbox.
+
```

Comment:
Change to --> If you have no workloads set up, you will be prompted to [create one](https://docs.newrelic.com/docs/new-relic-one/use-new-relic-one/workloads/use-workloads/) before you can use errors inbox.

Add --> Once you select your workload, your inbox should populate with error groups.